deploy/template.yaml

```diff
@@ -1568,4 +1568,4 @@ Outputs:
   CoreStubURL:
     Condition: IsDevOrBuild
     Description: "The core stub URL"
-    Value: !Sub "{{resolve:ssm:/check-hmrc-cri-api/smoke-tests/core-stub-url}}"
+    Value: !Sub "{{resolve:ssm:/check-hmrc-cri-front/core-stub-url}}"
```

Comment:
The `CoreStubURL` and `WebsiteHost` outputs don't belong in the template - it's only used in smoke tests so we shouldn't litter the production template with these values.

Since they're only used in smoke tests, the values can be hardcoded in the script or the script can pull them directly from SSM.